Diagnostics/AVTester/Test-ExchAVExclusions.ps1

```diff
@@ -360,6 +346,9 @@ while ($currentDiff -gt 0) {
                 # Remove Microsoft modules
                 $ProcessModules = $ProcessModules | Where-Object { $_.FileVersionInfo.CompanyName -ne "Microsoft Corporation." -and $_.FileVersionInfo.CompanyName -ne "Microsoft" -and $_.FileVersionInfo.CompanyName -ne "Microsoft Corporation" }
 
+                # Remove Oracle modules on FIPS
+                $ProcessModules = $ProcessModules | Where-Object { (($_.FileName -notlike "*\FIP-FS\Bin\*" -and $_.FileVersionInfo.CompanyName -ne "Oracle Corporation")) }
```

Comment:
Should we care if there are Oracle modules loaded into the process and those modules are _not_ in FIP-FS\Bin? Because this will exclude them no matter where they are.

![image](https://github.com/microsoft/CSS-Exchange/assets/4518572/3f18a601-f930-4900-9cca-f35da445c478)

Reply:
Correct, so the logic that we need to have is the following: 

```powershell
(-not($_.FileName -like "*\FIP-FS\Bin\*" -and $_.FileVersionInfo.CompanyName -eq "Oracle Corporation")) 
```

@iserrano76 please make this change and we can approve it.